### ISSUE TITLE
About send to server section

### ISSUE THREAD

**Issue by lingboo** [ThorGroup/FinalProject#8]
@ilentt did you done with crop image when take a picture or choose picture from gallery ? And i think, you need to let user choose only 1 category from that, just only 1. If see this, please reply

**Comment by perich** [comment]
@lingboo @ilentt Yes, Just one category. And there should only be a total of 4 or 5 choices. Let's decide now what they are. I think it should be: "food", "hotel/accomodation", "place of interest", "sight/view", "other". What do you think? Should we change this?

**Comment by ilentt** [comment]
@lingboo I am implement crop image feature, however it is crashed. I am workaround to resolve this issue. 

@perich About category, I will change to radio button instead of check box. If the team decided choose your suggested I will apply into next commit.

**Comment by lingboo** [comment]
yes, that's right. will radio button will only 1 choosen instead of multiple choosen, right ?

**Comment by ilentt** [comment]
Ok @lingboo

**Comment by ilentt** [comment]
@perich @lingboo I just update classify as your suggested
